src/main/resources/static/resources/testRules_constants.js

```diff
@@ -0,0 +1,42 @@
+// Short usage description of test rules
+var test_rule_info = '1. Load buttons are for loading rules/events from extern files.\n' +
+                    '    It is possible to choose if you want to append or replace\n' +
+                    '    aready writen rules/events in the text areas.\n'+
+                    '      * Upload rules/events must be in json list, ex. [{},{}].\n' +
+                    '\n2. By clicking on "Get Template" buttons you will download rules\n' +
+                    '    respective events template. Rule´s template contains 3 rules\n' +
+                    '    and event´s template contains 3 events.\n'+
+                    '\n3. Add buttons are for adding new rules/events. When you click\n' +
+                    '    on one of them, a new text area will show up.\n'+
+                    '      * There can be only one JSON object per text area. One\n' +
+                    '          rule/event per text area.\n' +
+                    '\n4. Download buttons are for downloading edited rules/events.\n'+
```

Comment:
Add to number 4 suggestion: This enables you to edit rules and or events locally and then upload them using above mentioned upload rules/events.

Reply:
Done